packages/governance/src/constants.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Enum of parameter types
+ *
+ * `Unknown` is an escape hatch for types we haven't added yet. If you are
+ * developing a new contract and use UNKNOWN, please also file an issue to ask
+ * us to support the new type.
+ */
+export const ParamTypes = /** @type {const} */ ({
+  AmountValue: 'amountValue',
+  Brand: 'brand',
+  BrandedAmount: 'brandedAmount',
+  BrandedRatio: 'brandedRatio',
+  Instance: 'instance',
+  Installation: 'installation',
+  Invitation: 'invitation',
+  Nat: 'nat',
+  RatioValue: 'ratioValue',
+  String: 'string',
+  Unknown: 'unknown',
```

Comment:
> javascript had the convention that the values of an enum were upper case while the enum itself was camel case

Like this?
```
  RatioValue: 'RATIO_VALUE'
```

That style is common, yeah. Used in https://www.typescriptlang.org/docs/handbook/enums.html

> This is gratuitously incompatible with the rest of our enums.

I don't know an official style but I see other enums like this:
```
  ACTIVE: 'active',
  LIQUIDATING: 'liquidating',
  CLOSED: 'closed',
  LIQUIDATED: 'liquidated',
  TRANSFER: 'transfer',
```

I'll change it (back) to that. I was going for more readability of this key but I'm changing my position to: no manifest _values_ for literals of _types_ space. That's out of scope of this PR so I'll UPPER_CASE now and work on https://docs.google.com/document/d/1JbZhNtfgZ7Q4O6rnpkN3fj9-Melc_SCyDLqOpY15Hp0/edit

Reply:
I meant what you ended up on.

Maybe you'd call those (what I meant by "values") the _Names_ in the enum? (ACTIVE, LIQUIDATING)  The strings or numbers seem like noise to me. They're not an interesting part of the enum to me. They should appear only once to satisfy the language, and every other reference should be by name. Otherwise the enum isn't saving anything.  I guess that's part of why I dislike using the strings in the code.

Reply:
Yeah I'd call those name or keys. I totally agree that the point of an enum is to hide away the underlying literals (which I'd call values, à la key-values)